src/dotnet/APIView/APIViewWeb/Controllers/PullRequestController.cs

```diff
@@ -30,14 +30,15 @@ public PullRequestController(PullRequestManager pullRequestManager, ILogger<Auto
             string filePath, 
             int pullRequestNumber, 
             string commitSha,
-            string repoName)
+            string repoName,
+            string packageName)
```

Comment:
This will work for most cases but something to consider for Java is that packageName is often artifactId and groupId. So when testing in Java we may hit a conflict but it should only ever happen if we mix track 1 and track 2 libraries but currently we only support track 2 so shouldn't be an issue. Just something to keep in the back of your mind.